parser.js

```diff
@@ -37,6 +37,9 @@ const canonicalUrlRules = buildRuleset('url', [
   ['link[rel="canonical"]', node => node.element.href],
 ]);
 
+const keywordsRules = buildRuleset('keywords', [
+  ['meta[name="keywords"]', node => node.element.content],
```

Comment:
Do we want to return the `content` as string value for the `<meta name="keywords" content="foo bar" />` tag, or return as an array of [trimmed] keywords?

I'd probably vote for the latter, plus, it's more inline with what Embedly proxy does:

```js
"keywords": [
  {
      "name": "charcoal",
      "score": 95
  },
  {
      "name": "lemon",
      "score": 63
  },
  {
      "name": "drink",
      "score": 32
  },
  {
      "name": "lemonade",
      "score": 32
  },
  {
      "name": "water",
      "score": 29
  },
  {
      "name": "juice",
      "score": 25
  },
  {
      "name": "detox",
      "score": 24
  },
  {
      "name": "toxins",
      "score": 21
  },
  {
      "name": "activated",
      "score": 20
  },
  {
      "name": "detoxifier",
      "score": 20
  }
],
```
```sh
$ http https://embedly-proxy.services.mozilla.com/v2/extract urls:='["http://yogimami.com/Empire/tag/homemade-charcoal-lemonade-juice/"]' -j -v
```